pkg/gofr/gofr_test.go

```diff
@@ -134,7 +142,7 @@ func TestGofr_ServerRun(t *testing.T) {
 	}
 
 	re, _ := http.NewRequestWithContext(context.Background(), http.MethodGet,
-		"http://localhost:"+strconv.Itoa(defaultHTTPPort)+"/hello", http.NoBody)
+		"http://localhost:"+strconv.Itoa(port)+"/hello", http.NoBody)
```

Comment:
It's fun you are alternatively use itoa and fmt.Sprint

Reply:
Sure changed to `"http://localhost:"+fmt.Sprint(port)`